src/components/modules/puzzles/puzzlesBanner/squares/ContentSquares.tsx

```diff
@@ -0,0 +1,232 @@
+import React from 'react';
+import { css, SerializedStyles } from '@emotion/core';
+import { headline } from '@guardian/src-foundations/typography';
+import { brandAlt, neutral } from '@guardian/src-foundations/palette';
+import { space } from '@guardian/src-foundations';
+import { from, until } from '@guardian/src-foundations/mq';
+import { Square } from './Square';
+import { qrCode } from '../images';
+
+function desktopGridPlacement(row: number, column: number) {
+    return css`
+        ${from.desktop} {
+            grid-row: ${row};
+            grid-column: ${column};
+        }
+    `;
+}
+
+// We can fake the 'gap' rule in IE11 by adding extra rows/columns to act as the gap
+// These gap rows/columns can also be flexible in ways that a standard gap rule can't
+function withIECompatibleGap(rowsOrCols: string[], gap: string) {
+    return rowsOrCols.join(` ${gap} `);
+}
+
+const boxShadow = '0px 6px 0px rgba(0, 0, 0, 0.25);';
+const border = `2px solid ${neutral[0]}`;
+
+const contentSquareSide = css`
+    position: relative;
+    background-color: ${neutral[20]};
+    border: ${border};
+    border-top: none;
+    border-right: none;
+    border-radius: 2px 0 0 0;
+    transform: translate(-10px, 2px);
+    height: 100%;
+    box-shadow: ${boxShadow};
+
+    ::before {
+        content: ' ';
+        display: block;
+        width: 6px;
+        background-color: ${neutral[20]};
+        border-top: ${border};
+        height: 100%;
+        transform: translateY(-2px) skewY(-30deg);
+    }
+`;
+
+const contentSquare = css`
+    z-index: 2;
+    ${headline.xxxsmall({ fontWeight: 'bold' })};
+    color: ${neutral[7]};
+    border-bottom: ${border};
+    padding-top: 0;
+    box-shadow: ${boxShadow};
+    min-width: ${space[24]}px;
+    min-height: ${space[24]}px;
+
+    p {
+        padding: ${space[1]}px;
+        padding-left: 0;
+        margin: 0;
+        margin-left: -2px;
+    }
+
+    ${from.tablet} {
+        width: 154px;
+        height: 154px;
+    }
+
+    ${from.desktop} {
+        ${headline.xxsmall({ fontWeight: 'bold' })};
+        width: 180px;
+        height: 180px;
```

Comment:
Is it worth setting the dimensions of the squares as variables so you only have to maintain half as many rows?

Reply:
Good idea!